meshroom/ui/qml/GraphEditor/NodeDocumentation.qml

```diff
@@ -22,17 +22,100 @@ FocusScope {
         ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
         clip: true
 
-        TextEdit {
-            width: parent.parent.width
-            height: parent.height
-
-            padding: 8
-            textFormat: TextEdit.MarkdownText
-            selectByMouse: true
-            selectionColor: activePalette.highlight
-            color: activePalette.text
-            text: node ? node.documentation : ""
-            wrapMode: TextEdit.Wrap
+        ColumnLayout {
+            id: nodeDocColumnLayout
+            property real keyColumnWidth: 10.0 * Qt.application.font.pixelSize
+
+            Component {
+                id: nodeInfoItem
+                Rectangle {
+                    color: activePalette.window
+                    width: parent.width
+                    height: childrenRect.height
+                    RowLayout {
+                        width: parent.width
+                        Rectangle {
+                            id: nodeInfoKey
+                            anchors.margins: 2
+                            color: Qt.darker(activePalette.window, 1.1)
+                            Layout.preferredWidth: nodeDocColumnLayout.keyColumnWidth
+                            Layout.minimumWidth: 0.2 * parent.width
+                            Layout.maximumWidth: 0.8 * parent.width
+                            Layout.fillWidth: false
+                            Layout.fillHeight: true
+                            Label {
+                                text: modelData.key
```

Comment:
The labels for the keys should be capitalized. Generally speaking, we try to avoid lowercase fields. So we should have "Author", "License", and so on, instead of using directly the keys.